psl/parser-database/src/walkers/enum.rs

```diff
@@ -46,6 +48,16 @@ impl<'db> EnumWalker<'db> {
         (0..self.ast_enum().values.len()).map(move |idx| self.walk((self.id, idx)))
     }
 
+    /// How fields are indented in the enum.
+    pub fn indentation(self) -> IndentationType {
+        IndentationType::default()
```

Comment:
Are we sure about this?

Reply:
Yeah per convo with @tomhoule yesterday. If it doesn't line up, it should just be a case of re-running the formatter. The future goal is to remove these functions from both enum and model etc and instead ask the editor for this information

Reply:
That is fine for me!